src/main/scala/ai/privado/languageEngine/ruby/tagger/source/IdentifierTagger.scala

```diff
@@ -27,12 +27,19 @@ import ai.privado.cache.RuleCache
 import ai.privado.model.{InternalTag, RuleInfo}
 import ai.privado.tagger.PrivadoParallelCpgPass
 import ai.privado.utility.Utilities.{addRuleTags, storeForTag}
-import io.shiftleft.codepropertygraph.generated.Cpg
+import io.shiftleft.codepropertygraph.generated.nodes.Call
+import io.shiftleft.codepropertygraph.generated.{Cpg, Operators}
 import io.shiftleft.passes.ForkJoinParallelCpgPass
-import io.shiftleft.semanticcpg.language._
+import io.shiftleft.semanticcpg.language.*
+
+import scala.jdk.CollectionConverters.CollectionHasAsScala
 
 class IdentifierTagger(cpg: Cpg, ruleCache: RuleCache) extends PrivadoParallelCpgPass[RuleInfo](cpg) {
 
+  val cachedCall: List[Call] = cpg.call
```

Comment:
Where is this used?

Reply:
Oh, this is not used anywhere. Will need to remove this. 
Will remove this in subsequent PR's as a new commit will again take time for comparison report